Project.toml

```diff
@@ -4,6 +4,8 @@ authors = ["rafaelorozco <rao9787@gmail.com>", "Mathias Louboutin <mathias.loubo
 version = "0.4.0"
 
 [deps]
+FFTResampling = "c3173eba-33fe-4761-8a84-0ec65b39ba9a"
+Images = "916415d5-f1e6-5110-898d-aaa5f9f070e0"
```

Comment:
Images is massive I would avoid it. If pad is all you need it's trivial to do it by hand